### PR TITLE
Thank you screen

### PR DESCRIPTION
**Pull request by DimaPylyp**
Created Thank You Screen and added needed constants.

### CODE REVIEW

**Comment by alexxcheung** [comment]
Thank you

File: LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+import UIKit
+import SafariServices
+
+class ThankYouViewController: UIViewController {
+    
+    private lazy var confettiView = SAConfettiView(frame: view.bounds)
+    
+    private let heartIcon : UIImageView = {
+
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift]: Remove space

File: LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+import UIKit
+import SafariServices
+
+class ThankYouViewController: UIViewController {
+    
+    private lazy var confettiView = SAConfettiView(frame: view.bounds)
+    
+    private let heartIcon : UIImageView = {
+        
+        let icon = UIImageView(image: UIImage(named: Images.heartIcon)!.withAlignmentRectInsets(UIEdgeInsets(top: -10, left: 0, bottom: 0, right: 0)))
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift]: make it in two lines
let icon = UIImageView()
icon.image = UIImage blablabla

File: LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        icon.contentMode = .scaleAspectFit
+        
+        icon.translatesAutoresizingMaskIntoConstraints = false
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift]: This is supposed to be in addConstraints method

File: LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        NSLayoutConstraint.activate([
+            icon.widthAnchor.constraint(equalToConstant: 60),
+            icon.heightAnchor.constraint(equalToConstant: 60)
+        ])
+        
+//        icon.layer.borderWidth = 1
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift]: Delete commented code

File: LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+    
+    
+    
+    private let titleLabel : UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 2
+        label.textAlignment = .left
+        label.font = UIFont.boldSystemFont(ofSize: 22)
+        label.text = LabelStrings.thankYou
+//        label.layer.borderWidth = 1
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift]: Delete commented code

File: LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+        stackView.axis = .vertical
+        stackView.spacing = 16
+        stackView.distribution = .equalCentering
+        stackView.alignment = .fill
+//        stackView.layer.borderWidth = 1
+//        stackView.layer.borderColor = UIColor.blue.cgColor
+        return stackView
+    }()
+    
+    private let contentView = UIView()
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift]: Bad copy pasting

File: LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(closeVC), name: Notification.Name(rawValue: "WillTransition"), object: nil)
+    }
+    
+    @objc private func didTapContinue() {
+        print("tap")
+        defaults.set(true, forKey: Keys.onboardingComplete)
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift]: No, this is not onboarding

File: LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift [hunk 8]
```diff
@@ -0,0 +1,167 @@
+//
+//  ThankYouViewController.swift
+//  LetsDinnerV2 MessagesExtension
+//
+//  Created by DiMa on 15/04/2020.
+//  Copyright © 2020 Eric Ordonneau. All rights reserved.
+//
+
+import UIKit
+import SafariServices
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift]: Why SafariServices?

**Review by Ericordo — APPROVED** [review]

**Pull request merged**